python/ray/util/placement_group.py

```diff
@@ -242,7 +242,7 @@ def check_placement_group_index(placement_group: PlacementGroup,
             raise ValueError("If placement group is not set, "
                              "the value of bundle index must be -1.")
     elif bundle_index >= placement_group.bundle_count \
-            or bundle_index < -1:
+        or bundle_index < -1:
```

Comment:
Can you indent here back again? Is it formatted automatically? 

If or is aligning with the line below, it looks confusing.

Reply:
Is this github display problem?  my local display is right.